### PR TITLE
Updating engineering action playbook 

### PR DESCRIPTION
**Pull request by tracom1**
This has a lot of proposed changes that the TAC needs to review, so I'd like them to see what the diff is easily.

(Thus why I'm proposing a merge to staging).

I'm also wondering if we should adopt the story module for this too since we're here @andrewnhem ?

### CODE REVIEW

**Comment by andrewnhem** [comment]
@tracom1 Nicely done. Yeah I think if the crew has added more of a narrative to the piece, which was formerly ALL user stories, transforming the stories content into the proper format is required. Do you need help with this or are you good?

File: _stories/anomaly-detection.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ tags:
 persona:
 stage:
 company:
+order: 1
```
Comment (andrewnhem) [inline _stories/anomaly-detection.md:11]: @tracom1 the module is in the latest version of this branch. Have you tested this out yet? I think we just need to get the stories in the order that you'd like, merge to staging, and you'll have a nice candidate to present to TAC.

File: projects/encouraging-engineers-to-take-action.md [hunk 2]
```diff
@@ -1,93 +1,213 @@
...
 
 How finance, operations, and executives can work better together at encouraging more cloud cost optimization from their engineering teams.
 
 ## Before you begin
+
 You should understand the basics of how cloud computing works, know the key services on your cloud providers, including their common use cases, and have a basic understanding of billing and pricing models. Being able to describe the basic value proposition of running in the cloud and understand the core concept of using a pay-as-you-go consumption model are also necessary.
 
 You’ll also need to have a base level of knowledge of at least one of the three main public cloud providers (AWS, Azure, Google Cloud). For AWS, we recommend AWS Business Professional training or, even better, the AWS Cloud Practitioner certification. For Google, check out the Google Cloud Platform Fundamentals course. For Azure, try the Azure Fundamentals learning path. Each can usually be completed in a full day workshop.
 
 ## Relevant FinOps Framework components
```
Comment (andrewnhem) [inline projects/encouraging-engineers-to-take-action.md:25]: @tracom1 We're going to need to re-do this once the Framework gets to where it needs to be. I think these will still resolve properly with the way things are set up today, but just heads up that there's going to be rework here (and in other Projects).

**Review by andrewnhem — CHANGES_REQUESTED** [review]
@tracom1 @stevetrask Nudge here to keep this going to get it over to staging and to its voting.

**Review by rileyjenk — DISMISSED** [review]

File: projects/encouraging-engineers-to-take-action.md (hunk 2 above)
Comment (tracom1) [reply projects/encouraging-engineers-to-take-action.md:25]: That's future Tracy and Andrew's problem!

**Review by cccp7171 — APPROVED** [review]

**Review by andrewnhem — APPROVED** [review]

**Comment by stevetrask** [comment]
Looks good!

**Pull request merged**